combat-trainer.lic

```diff
@@ -333,6 +333,15 @@ class LootProcess
     echo("  @last_rites: #{@last_rites}") if $debug_mode_ct
     @last_rites_timer = Time.now - 600
 
+    if settings.box_loot_limit
```

Comment:
I _think_ the `if` is unnecessary. In the 'missing' case, we assign them to `nil` which is what they would be if unassigned.

Reply:
might be weird, but id rather it not do the rummaging if this setting isnt being used.